ui/apps/platform/src/utils/ensure.ts

```diff
@@ -12,3 +13,55 @@ export function ensureString(value: unknown): string {
     }
     return '';
 }
+
+/**
+ * Helper function to ensure that the provided value is converted to an array of strings.
+ *
+ * Example:
+ * ensureStringArray(['a', 'b', 'c']);   // returns ['a', 'b', 'c']
+ * ensureStringArray('hello');           // returns ['hello']
+ * ensureStringArray([1, 'b', {}]);      // returns ['b'] (filters out non-string elements)
+ * ensureStringArray(123);               // returns []
+ *
+ * @param value - The input value, which could be of any type.
+ * @returns {string[]} - If the value is an array, it filters and returns only the string elements.
+ *                       If the value is a single string, it wraps it in an array and returns it.
+ *                       For all other values, it returns an empty array.
+ */
+export function ensureStringArray(value: unknown): string[] {
+    if (Array.isArray(value)) {
+        const result: string[] = value.filter((element) => typeof element === 'string');
```

Comment:
For primary use case to sanitize parse data from query string, `filter` does seem like an improvement over `every` in original version of this function.

Reply:
➕